File: python/codeeval/173_without_repetitions.py
import sys

def without_rep(inp):
    l = None
    ret = ''
    for ch in inp:
        if ch != l:
            ret += ch
            l = ch
    return ret

with open(sys.argv[1], 'r') as test_cases:
    for test in test_cases:
        print(without_rep(test))

File: javascript/codeeval/173_without_repetitions.js
var fs  = require("fs");

function withoutRep(inp) {
    var l;
    var ret = '';
    for (var i = 0; i < inp.length; i++) {
        if (inp[i] != l) {
            ret += inp[i]
            l = inp[i]
        }
    }
    return ret
}

fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line != "") {
        console.log(withoutRep(line))
    }
});